lib/teiserver_web/controllers/admin/user_controller.ex

```diff
@@ -953,17 +953,37 @@ defmodule TeiserverWeb.Admin.UserController do
   def relationships(conn, %{"id" => id}) do
     user = Account.get_user!(id)
 
+    relationships = %{
+      friends: Account.list_friend_ids_of_user(id),
+      friend_requests: Account.list_incoming_friend_requests_of_userid(id),
+      followed: Account.list_userids_followed_by_userid(id),
+      ignored: Account.list_userids_ignored_by_userid(id),
+      avoided: Account.list_userids_avoided_by_userid(id),
+      avoiding: Account.list_userids_avoiding_this_userid(id),
+      blocked: Account.list_userids_blocked_by_userid(id),
+      blocking: Account.list_userids_blocking_this_userid(id)
+    }
+
     user_ids =
-      (user.data["friends"] ++ user.data["friend_requests"] ++ user.data["ignored"])
+      relationships
+      |> Map.values()
+      |> List.flatten()
       |> Enum.uniq()
 
+    user_ids
+    |> IO.inspect(limit: :infinity)
```

Comment:
I suspect you meant to remove this, there's one a few lines later on 981 too.

Reply:
Yea this is still draft, work in progress.